doc/getting_started/index.rst

```diff
@@ -246,6 +246,33 @@ directory using west:
          cd zephyrproject
          west update
 
+.. rst-class:: numbered-step
+
+Export Zephyr CMake package
+***************************
+
+The :ref:`cmake_pkg` can be exported to CMake's user package registry.
```

Comment:
"can" implies "if you want/need to", but from reading this, I don't understand why I want/need to do this. A short sentence like "This is relevant if you ..." would be very useful here.

Reply:
Good point.
Will try to see if I can rephrase.

You can decide not to use this feature, just as you can decide not to use `west`, but both of them makes life much easier :wink:

Reply:
Try to make it more starter friendly